pallets/liquidation/src/lib.rs

```diff
@@ -220,11 +221,12 @@ impl<T: Config> Pallet<T> {
 
     fn transform_account_collateral(
     ) -> Result<BTreeMap<T::AccountId, (FixedU128, Vec<CollateralMisc>)>, Error<T>> {
-        let result = pallet_loans::AccountCollateral::<T>::iter().fold(
+        let result = pallet_loans::AccountDeposits::<T>::iter().fold(
             BTreeMap::<T::AccountId, (FixedU128, Vec<CollateralMisc>)>::new(),
-            |mut acc, (k1, k2, ctoken_balance)| {
+            |mut acc, (k1, k2, deposits)| {
```

Comment:
can we give k1, k2 better meaningful name

Reply:
Ok, I will fix it while I deal with the new TODO items added in the liquidation pallet.